nautobot/extras/models/models.py

```diff
@@ -366,6 +370,8 @@ def get_absolute_url(self):
 )
 class ExportTemplate(BaseModel, ChangeLoggedModel, RelationshipModel, NotesMixin):
     # An ExportTemplate *may* be owned by another model, such as a GitRepository, or it may be un-owned
+    # FIXME(timizuo): Related_name cannot be updated to `export_templates` as it collides with
+    #  ExportTemplate.owner_content_type related name
```

Comment:
I don't think this is a FIXME per se - think it should just be left as-is for the `owner_content_type` field.